docs/design/fabric-v2+/fpc-management.md

```diff
@@ -59,3 +59,32 @@ indicating that the chaincode enclave is ready to endorse transaction proposals.
 
 Key distribution commands are not supported in the initial version of FPC.
 Users must aware that each FPC chaincode runs in a single enclave at a designated peer.
+
+
+## FPC Endorsement Policies
+
+The initial version of FPC limits the use for endorsement policies and currently
+supports a designated endorser policies only. That is, only a single endorsing peer is
+responsible to execute transactions of a FPC chaincode for the consortium.  As a FPC
+chaincode is protected by the use of Trusted Execution Technology, from a security
+perspective, the integrity guarantees gained from the endorsement model are similar.
```

Comment:
similar to the integrity guarantee in a typical Fabric system with multiple Peer endorsements.